libnmstate/nm/bridge.py

```diff
@@ -165,36 +176,57 @@ def _get_bridge_setting(nmdev):
     return bridge_setting
 
 
-def _get_bridge_ports_info(port_profiles, vlan_filtering_enabled=False):
-    ports_info = []
-    for p in port_profiles:
-        port_info = _get_bridge_port_info(p)
+def _get_bridge_ports_info(
+    port_names_profiles, port_runtime_names, vlan_filtering_enabled=False
+):
+    ports_name_info = {}
```

Comment:
I do not understand the name.. it seems to be `ports_info_by_name`.

Reply:
Done, thanks!